pkg/minikube/console/console.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package console provides a mechanism for sending localized, stylized output to the console.
+package console
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+	isatty "github.com/mattn/go-isatty"
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+)
+
+// By design, this package uses global references to language and output objects, in preference
+// to passing a console object throughout the code base. Typical usage is:
+//
+// console.SetOutFile(os.Stdout)
+// console.Out("Starting up!")
+// console.OutStyle("status-change", "Configuring things")
+
+// console.SetErrFile(os.Stderr)
+// console.Fatal("Oh no, everything failed.")
+
+// NOTE: If you do not want colorized output, set MINIKUBE_IN_COLOR=0 in your environment.
+
+var (
+	// outFile is where Out* functions send output to. Set using SetOutFile()
+	outFile fdWriter
+	// errFile is where Err* functions send output to. Set using SetErrFile()
+	errFile fdWriter
+	// preferredLanguage is the default language messages will be output in
+	preferredLanguage = language.AmericanEnglish
+	// our default language
+	defaultLanguage = language.AmericanEnglish
+	// useColor is whether or not color output should be used, updated by Set*Writer.
+	useColor = false
+	// OverrideEnv is the environment variable used to override color/emoji usage
+	OverrideEnv = "MINIKUBE_IN_COLOR"
```

Comment:
this should be a const rather

Reply:
I intentionally left this as a var, in case someone else wants to abuse this library for their own nefarious uses.